### ISSUE TITLE
Show default hotkeys in json

### ISSUE THREAD

**Issue by LowWeiLin** [HubTurbo/HubTurbo#884]
Show the default hotkey values in the json file so users know what they can change.

**Comment by lejolly** [comment]
The current implementation detects and offers to reset to the defaults if there are missing shortcut values. If there are no shortcut values defined, no error is shown to the user but it writes the default shortcuts to `global.json` on exit. Is this good enough?

**Comment by dariusf** [comment]
I guess the issue is that the defaults only appear after the first exit. If a user opens the json file right after opening HT, the shortcuts won't be there yet.

**Comment by lejolly** [comment]
yup, but neither will any information be present. all default fields like last viewed repo, credentials, etc are all empty too.

**Comment by dariusf** [comment]
Those fields store data about the session, though, and are not meant to be changed by users. Only saved boards might be tweaked manually. Maybe we should separate these, and take special care to fill (and possibly reload) fields which the user is supposed to change.

**Comment by damithc** [comment]
This is what I mentioned y'day. Session/computer specific data should be separated from user-specific settings so that the latter can be synced to multiple machines.
